server/routes/v1/model.ts

```diff
@@ -121,7 +121,7 @@ export const getModelVersions = [
       throw NotFound({ code: 'model_not_found', uuid }, `Unable to find model '${uuid}'`)
     }
 
-    const versions = await findModelVersions(req.user, model._id, { thin: true })
+    const versions = await findModelVersions(req.user, model._id, { thin: true, showLogs: true })
```

Comment:
Is this intentionally hard-coded to `true` or should we do something like `const showLogs = logs === 'true'`?

Reply:
Oh no this one slipped under the radar! Good spot